### PR TITLE
New mod C17-EXT and C17-EXT-CARTRIDGE

### PR DESCRIPTION
**Pull request by Obomaker3D**
#### Which mods/configurations/profiles are added by this PR?
C17-EXT: 
The C17-EXT filter replaces the standard filter of the Voron 2.4 and to add one on the Switchwire . It has a 120mm fan and can use the C17-EXT-Cartridge filter (HEPA 13 and Carbon) or commercial HEPA Carbon filters.

C17-EXT-CARTRIDGE: 
The C17-EXT-CARTRIDGE is designed to be used with the C17-EXT HEPA Carbon filter

#### How was it tested? 

It has been tested on a Voron 2.4 serial V2.3526 but not on the Voron Switchwire. The documentation has been shared on FaceBook for review. No other people have tested it.

#### Any background context you want to provide?

I made a PDF documentation in VoronDesign format, specifying on the first page that it was not made by the VoronDesign team. I hope this is not a problem.

### CODE REVIEW

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The list of parts is described in the manual.
+
+## HEPA 13 Filter
+
+Use HEPA 13 filters from 10mm to 18mm thick. The best is to find a HEPA 13 filter for vacuum cleaner and cut it with a cutter. The references change according to the country.
+
+I tested this one after cutting: 
+
+- The best but I do not find the reference in other countries: [Amazon.fr](https://www.amazon.fr/gp/product/B01MCTSXVR/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1)
```
Comment (cbc02009) [inline printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md:25]: please shorten to `https://www.amazon.fr/gp/product/B01MCTSXVR` the rest is just tracking data.

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ 
+ - [Aliexpress 1](https://fr.aliexpress.com/item/4000071269334.html?spm=a2g0o.productlist.0.0.574c118d9Bmk6H&algo_pvid=f6bc77a3-4082-4ada-840b-ccc360a0eb8a&aem_p4p_detail=202205140919487673921735755640028568333&algo_exp_id=f6bc77a3-4082-4ada-840b-ccc360a0eb8a-4&pdp_ext_f=%7B%22sku_id%22%3A%2210000000183120618%22%7D&pdp_npi=2%40dis%21EUR%21%219.53%21%21%211.99%21%21%402100bdec16525451882826492ee768%2110000000183120618%21sea)
```
Comment (cbc02009) [inline printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md:27]: please change to `https://www.aliexpress.com/item/4000071269334.html` everything after is just tracking data.
also please change to www from fr, aliexpress has a weird bug if you go to another reigon's site you have to manually delete cookies to ever be able to see your region's site ever again.

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- [Aliexpress 2](https://fr.aliexpress.com/item/4001298732898.html?spm=a2g0o.order_list.0.0.21ef5e5bHvenA3&gatewayAdapt=glo2fra)
```
Comment (cbc02009) [inline printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md:29]: same comments as above.

File: printer_mods/README.md [hunk 4]
```diff
@@ -282,6 +282,8 @@ like so:
 |Oakman |[FYSETC Spider Metal DIN Mount](./Oakman/FYSETC_Spider_Metal_DIN_Mount) |Use a metal DIN mount for your FYSETC Spider |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
 | |[BTT 24V UPS Metal DIN Mount](./Oakman/BTT_24V_UPS_Metal_DIN_Mount) | Adapter to use a metal DIN mount for the BTT 24V UPS |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
 | |[RPI Metal DIN Mount](./Oakman/RPI_Metal_DIN_Mount) | Adapter for metal DIN mounting the Raspberry Pi |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
+|OboMaker3D |[C17-EXT Filter HEPA Carbon](./OboMaker3D/C17-EXT) |Replaces standard filter with new HEPA Crabon and 120mm fan |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (cbc02009) [inline printer_mods/README.md:285]: there should be 5 columns here for the 5 printers. not sure which one is missing.

File: printer_mods/README.md [hunk 5]
```diff
@@ -282,6 +282,8 @@ like so:
...
+| |[C17-EXT-CARTRIDGE for C17-EXT filter](./OboMaker3D/C17-EXT-CARTRIDGE) | Alternative to a commercial cartridge for the C17-EXT filter |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (cbc02009) [inline printer_mods/README.md:286]: same as above.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
4c_Filter_Box_Grid_Extract_100mm.stl, 4b_Filter_Box_Grid_Extract_125mm.stl, and 1_Filter_Box.stl appear to be upside down.

**Comment by Obomaker3D** [comment]
All corrections done

File: printer_mods/README.md (hunk 5 above)
Comment (Obomaker3D) [reply printer_mods/README.md]: Done

File: printer_mods/README.md (hunk 4 above)
Comment (Obomaker3D) [reply printer_mods/README.md]: Done

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md (hunk 3 above)
Comment (Obomaker3D) [reply printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md]: Done

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md (hunk 2 above)
Comment (Obomaker3D) [reply printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md]: Done

**Comment by Obomaker3D** [comment]
All corrections done, thank you for your work

**Comment by Obomaker3D** [comment]
> 4c_Filter_Box_Grid_Extract_100mm.stl, 4b_Filter_Box_Grid_Extract_125mm.stl, and 1_Filter_Box.stl appear to be upside down.

Done

**Comment by Obomaker3D** [comment]
> 4c_Filter_Box_Grid_Extract_100mm.stl, 4b_Filter_Box_Grid_Extract_125mm.stl, and 1_Filter_Box.stl appear to be upside down.

Did I make a mistake or are you unavailable? Thank you for your work

File: printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## HEPA 13 Filter
+
+Use HEPA 13 filters from 10mm to 18mm thick. The best is to find a HEPA 13 filter for vacuum cleaner and cut it with a cutter. The references change according to the country.
+
+I tested this one after cutting: 
+
+- The best but I do not find the reference in other countries: [Amazon.fr](https://www.amazon.fr/gp/product/B01MCTSXVR)
+ 
+- [Aliexpress 1](https://fr.aliexpress.com/item/4000071269334.html)
```
Comment (cbc02009) [inline printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md:27]: Please change these to https://www.aliexpress.com aliexpress does weird stuff if you go to the wrong countries' site.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Yes, sorry. I've been very busy the last couple of weeks. The stl rotation looks good, just need a minor change to the readme.

**Comment by Obomaker3D** [comment]
Thanks for the work, this is my first pull request, I was afraid I made a mistake.

**Comment by Obomaker3D** [comment]
Hello, I worked hard for this mon, thank you to indicate me if it is a problem to the group for its validation? I am well aware that you must have a lot of requests/work, but you are connected and for 20 days you have not validated while the last correction was minor.
If there is a problem please tell me, if it is a time problem, ok.

**Comment by cbc02009** [comment]
Sorry, it's 100% a time problem. Got a lot of stuff going on right now, so I haven't even had time to look at the pull requests in a couple weeks. I'll see what I can do this weekend, but no promises

**Comment by Obomaker3D** [comment]
The delay is not a problem if it is only your time, you are a volunteer, it is precious.
I am worried that my documentation and its form close to the 2.4 documentation is a problem

**Review by cbc02009 — APPROVED** [review]

**Comment by Obomaker3D** [comment]
thank you, no more action from my side?

**Comment by cbc02009** [comment]
@Obomaker3D The CI tests seem to be stuck. can you push a simple commit that adds an extra space or newline to one of the files just to make the CI run again?

**Comment by Obomaker3D** [comment]
> @Obomaker3D The CI tests seem to be stuck. can you push a simple commit that adds an extra space or newline to one of the files just to make the CI run again?

Done

**Comment by MapleLeafMakers** [comment]
CAD Files are missing.  All submissions require CAD sources in either STEP or SCAD format.   This pull request was opened before the requirement started being enforced, but unfortunately it's still a requirement.

**Comment by Obomaker3D** [comment]
> CAD Files are missing. All submissions require CAD sources in either STEP or SCAD format. This pull request was opened before the requirement started being enforced, but unfortunately it's still a requirement.

Hello, thank you for your work. On the other hand, I searched for this rule on VoronDesign and Git and I can't find it. Can you tell me where it is specified? 
Can you give me the information why this rule was added and confirm that you do not represent a private company? I am very attached to the idea of a free and collaborative system.
Thanks for everything.

**Comment by newphreak** [comment]
https://github.com/VoronDesign/VoronUsers/blob/master/.github/PULL_REQUEST_TEMPLATE.md
The template was updated the 1st of april this year.
image
The rule was added so that if a mod becomes outdated and the original author disappears, the torch can be given to someone else and update it. After a given time period etc. If I'm not fully mistaken.

**Comment by Obomaker3D** [comment]
> https://github.com/VoronDesign/VoronUsers/blob/master/.github/PULL_REQUEST_TEMPLATE.md
> The template was updated the 1st of april this year.
> image
> The rule was added so that if a mod becomes outdated and the original author disappears, the torch can be given to someone else and update it. After a given time period etc. If I'm not fully mistaken.

Done, thanks for your time

**Comment by MapleLeafMakers** [comment]
@Obomaker3D CAD Files need to be .STEP or .SCAD format.  You've submitted f3d files.  You may include .f3d *in addition* to the required STEP or SCAD.  Fusion360 can export step files for you.

**Comment by MapleLeafMakers** [comment]
- There are empty readme.md files in the CAD folders of each mod which should be removed
- license.html files should be removed.  All mods are licensed under the GPLv3

**Comment by Obomaker3D** [comment]
> * There are empty readme.md files in the CAD folders of each mod which should be removed
> * license.html files should be removed.  All mods are licensed under the GPLv3

Ah ah, I do, but notice that the 2.4 model is published in f3d ... anything else my friend?

**Comment by cbc02009** [comment]
I have a feeling 7_join.stl is going to need fixing since it's showing up weird in the github renderer, but we'll see what the CI says.

Thank you for your patience @Obomaker3D we should get this finished very soon :)

**Pull request merged**